packages/authenticator/amplify_authenticator/lib/src/state/authenticator_state.dart

```diff
@@ -20,7 +19,7 @@ typedef BlocEventPredicate = bool Function(AuthState state);
 ///
 /// Intended to be used within custom UIs for the Amplify Authenticator.
 class AuthenticatorState extends ChangeNotifier {
-  AuthenticatorState(this._authBloc) {
+  AuthenticatorState(this._authBloc, {this.defaultDialCode = DialCode.us}) {
```

Comment:
Should be `required` with no default. We don't want to accidentally not pass this.